### ISSUE TITLE
Community stewardship of this package?

### ISSUE THREAD

**Issue by brylie** [TsumiNa/meteor-aurelia#10]
@TsumiNa thanks for this great initial work to bring the elegance of Aurelia to the Meteor community. I would like to 'revive' this package, to update it to use recent versions and methods for packaging such as NPM. Would you please help me understand what this package does, so that I can try my hand at creating an updated version?

**Comment by TsumiNa** [comment]
Thank you @brylie. Be honestly, I doesn't use Meteor for a long time and need someone like you to bring Aurelia to new Meteor. 😃 

This package is simple. It use systemjs(meteor port which I made it) to load Aurelia in front-end. Because Meteor use it's own html template system(blaze-html-templates), so I removed blaze-html-templates and spacebars packages  and use my own to compile html files with aurelia. The compiler is part of systemjs so here is nothing different from others did without meteor. You can see [systemjs](https://github.com/systemjs/systemjs) on github for details.

I heard that new Meteor has using normal npm to manage packages and have a systemjs like ES6 loader system to handle ES6 javascript. so I think this package should not be needed anymore.

Any questions please let me know, thank you!

**Comment by brylie** [comment]
OK, cool. So it might be possible to use the Aurelia NPM packages directly?

**Comment by TsumiNa** [comment]
@brylie Yes, I think so. Try it read Meteor package's tutorial to get more.

**Comment by nilsi** [comment]
Hi guys, How did it go? did you get aurelia to play with meteor nicely @brylie?

**Comment by brylie** [comment]
I have not worked directly with Meteor and Aurelia yet. I am still learning Aurelia on side projects, but seriously considering it for two existing open source projects. Hence the initial research on integrating the two.
